### PR TITLE
Changed the get_agent func logic to base on the platform only

### PR DESCRIPTION
**Pull request by ShacharKidor**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
relates: [CRTX-77146](https://jira-hq.paloaltonetworks.local/browse/CRTX-77146).

## Description
As discussed in this issue [CRTX-77146](https://jira-hq.paloaltonetworks.local/browse/CRTX-77146), the `demisto.demistoVersion()` function should return platform='x2' for XSIAM machines, and platform='xsoar' for XSOAR machines (both on-prem and cloud). 
Therefore, we can differentiate between XSIAM to XSOAR based on the platform info only (no need to use the version). 
In this PR I have changed that logic. 

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [x] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
@ShahafBenYakir  - Please let me know if you think this logic change should affect other integrations. If yes I will change that in them as well.

File: Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py [hunk 1]
```diff
@@ -21,9 +21,10 @@ def get_agent():
         within XSOAR (both on-prem and cloud).
         """
         platform = get_demisto_version().get('platform')
-        if platform == 'x2':
-            return 'xsoartim' if is_demisto_version_ge('8.0.0') else 'xdr'
-        return 'xsoartim'
+        if platform == 'x2':  # XSIAM
+            return 'xdr'
+        else:  # XSOAR (on-prem or cloud)
+            return 'xsoartim'
```
Comment (ShahafBenYakir) [inline Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py:27]: simpler not mandatory tho
```suggestion
        return 'xdr' if platform == 'x2' else 'xsoartim':  # XSIAM
```

File: Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_17.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Palo Alto Networks WildFire Reports
+
+- Fixed an issue where the integration failed to send the right agent parameter when used by XSIAM or XSOAR Cloud users.
```
Comment (ShahafBenYakir) [inline Packs/Palo_Alto_Networks_WildFire/ReleaseNotes/2_1_17.md:6]:
```suggestion
- Fixed an issue where the integration failed to send the right agent parameter when used by XSIAM users.
```

**Review by ShahafBenYakir — APPROVED** [review]
Nice

File: Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py (hunk 1 above)
Comment (ShacharKidor) [reply Packs/Palo_Alto_Networks_WildFire/Integrations/WildFireReports/WildFireReports.py:27]: Added, thanks.

**Comment by ShacharKidor** [comment]
This PR will be merged once the server-side fix of this issue [CRTX-77146](https://jira-hq.paloaltonetworks.local/browse/CRTX-77146) will be merged.

**Review by ShahafBenYakir — APPROVED** [review]

**Pull request merged**